src/senselab/audio/data_structures/audio.py

```diff
@@ -83,9 +108,82 @@ def from_filepath(cls, filepath: str | os.PathLike, metadata: Dict = {}) -> "Aud
         if not os.path.exists(filepath):
             raise FileNotFoundError(f"File {filepath} does not exist.")
 
-        array, sampling_rate = torchaudio.load(filepath)
+        if offset_in_sec < 0:
+            raise ValueError("Offset must be a non-negative value.")
+        if duration_in_sec < -1:
```

Comment:
so this should be something akin to:
`duration_in_sec == -1 or duration_in_sec > 0` (presumably they can't create a 0 second audio

Reply:
the issue with the current implementation is any value between -1 and 0 is valid